autoextract/request.py

```diff
@@ -12,19 +12,27 @@ class Request:
     A single request data for AutoExtract.
     See https://doc.scrapinghub.com/autoextract.html#requests
 
-    Note that articleBodyRaw is set to false by default here; API itself
-    defaults to true. Set articleBodyRaw=None to remove articleBodyRaw
+    Note that `articleBodyRaw` is set to false by default here; API itself
+    defaults to true. Set `articleBodyRaw=None` to remove `articleBodyRaw`
     parameter from the request and use server default.
+
+    You can override API params passing a dictionary with extra data using the
+    `extra` argument. Note that it will overwrite any previous configuration
+    made using standard attributes like `articleBodyRaw` and `fullHtml`.
     """
     url = attr.ib()  # type: str
     pageType = attr.ib()  # type: str
 
     meta = attr.ib(default=None)  # type: Optional[str]
     articleBodyRaw = attr.ib(default=False)  # type: Optional[bool]
-    fullHtml = attr.ib(default=None)  # type: Optional[bool]
+    fullHtml = attr.ib(default=False)  # type: Optional[bool]
```

Comment:
hey! What's the reason to change it to False?

Reply:
No one 🤦🏻 
It was made False by default on our other pull request but thinking about it we don't need to pass anything to the query because this is already the default value on the API.
Fixed!

Reply:
@victor-torres I see that you've rewritten the history of this PR - do you think you can change the commit message as well? It still says that fullHtml default value is changed to False.

Reply:
@kmike, I'm sorry but I have just merged this pull request. 

I missed your message somehow as it only appeared on my screen after hitting the merge button (out of sync issues?).

Anyway, should I try to fix it on master?

Reply:
@victor-torres I think it is better not to rewrite history on master. Let's keep it as-is.

Reply:
Agree.